examples/standard_platformer/lib/main.dart

```diff
@@ -9,40 +9,57 @@ import 'package:leap_standard_platformer/player.dart';
 import 'package:leap_standard_platformer/welcome_dialog.dart';
 
 void main() {
-  runApp(GameWidget(game: ExamplePlatformerLeapGame()));
+  runApp(
+    GameWidget(
+      game: ExamplePlatformerLeapGame(),
+    ),
+  );
 }
 
 class ExamplePlatformerLeapGame extends LeapGame
-    with HasTappables, HasKeyboardHandlerComponents {
+    with TapCallbacks, HasKeyboardHandlerComponents {
   late final Player player;
   late final SimpleCombinedInput input;
 
   @override
   Future<void> onLoad() async {
     await super.onLoad();
-
-    await loadWorldAndMap('map.tmx', 16);
-    setFixedViewportInTiles(32, 16);
+    await loadWorldAndMap(
+      tileSize: 16,
+      tiledMapPath: 'map.tmx',
+      tileCameraWidth: 32,
+      tileCameraHeight: 16,
+    );
 
     input = SimpleCombinedInput();
     add(input);
+
     player = Player();
-    add(player);
-    camera.followComponent(player);
+    cameraComponent.world?.add(player);
```

Comment:
We should just reference the `camera` field instead of creating a new `CameraComponent`, as of v1.9.0 it is created by default as a `CameraComponent`